locals.tf

```diff
@@ -7,6 +7,7 @@ locals {
       lokiURL         = "http://${local.fullnameOverride}-distributor.${var.namespace}:3100/loki/api/v1/push"
       labels          = {}
       grafanaAgentTag = "main-4f86002"
+      affinity        = {}
```

Comment:
From what I understood, you add this parameter here but you do not add a Terraform variable to set the affinities because you set them through the `helm_values`, am I right?

Reply:
Thanks for your comments, yes the affinity will be set through helm_values and not by variable.